SPECS/calamares/serialize-read-access.patch

```diff
@@ -1,43 +1,56 @@
-diff -ruN a/src/modules/partition/core/PartitionModel.cpp b/src/modules/partition/core/PartitionModel.cpp
---- a/src/modules/partition/core/PartitionModel.cpp	2023-01-27 14:12:11.942298970 -0800
-+++ b/src/modules/partition/core/PartitionModel.cpp	2023-01-27 14:12:06.786346392 -0800
-@@ -79,6 +79,7 @@
+From dda7d24e07ae2272731676be4be0b4e3567fe0f7 Mon Sep 17 00:00:00 2001
+From: Sam Meluch <sammeluch@microsoft.com>
+Date: Tue, 23 Jan 2024 16:00:27 -0800
+Subject: [PATCH] serialized read patch
+
+---
+ src/modules/partition/core/PartitionModel.cpp | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/src/modules/partition/core/PartitionModel.cpp b/src/modules/partition/core/PartitionModel.cpp
+index a9d49dc..fbd8bb5 100644
+--- a/src/modules/partition/core/PartitionModel.cpp
++++ b/src/modules/partition/core/PartitionModel.cpp
+@@ -74,6 +74,7 @@ PartitionModel::columnCount( const QModelIndex& ) const
  int
  PartitionModel::rowCount( const QModelIndex& parent ) const
  {
-+    QMutexLocker lock(&m_lock);
++    QMutexLocker lock( &m_lock );
      Partition* parentPartition = partitionForIndex( parent );
      if ( parentPartition )
-         return parentPartition->children().count();
-@@ -89,6 +90,7 @@
+     {
```

Comment:
It's easier to see when viewing the file as a side-by-side or just looking at it straight on, https://github.com/microsoft/azurelinux/blob/sammeluch/qt-upgrade/SPECS/calamares/serialize-read-access.patch.

Since the patch was regenerated, some of the brackets in the original source moved lines. We are not actually adding these open brackets, but rather they are an existing part of the code which the patch file uses to place the add/removes correctly.